domains/ahmed.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Ahmed's personal site",
+  "repo": "https://github.com/Dropheart",
+  "owner": {
+    "username": "dropheart",
+    "email": "ahmed@techfromtheheart.com"
+  },
+  "record": {
+    "CNAME": "techfromtheheart.com"
```

In the pr description you mentioned wanting a redirection. If that's the case you should use the URL record which will redirect `ahmed.is-a.dev` to `techfromtheheart.com`.

```suggestion
    "URL": "https://techfromtheheart.com"
```

If you meant to use a CNAME record, I can merge it in.

---

Oh, yes, sorry. I've changed it now- DNS things still sometimes confuse me, need to get better with it 😅